docs/building_the_project_with_dpcpp.rst

```diff
@@ -0,0 +1,446 @@
+.. _building_the_project_with_dpcpp:
+
+Building the Project with DPC++
+===============================
+
+This page describes building the oneMKL interface library with either the Intel(R) oneAPI DPC++ Compiler or open-source oneAPI DPC++ Compiler.
+For guidance on building the project with AdaptiveCpp, see :ref:`building_the_project_with_adaptivecpp`.
+
+.. _build_setup_with_dpcpp:
+
+Build Setup
+###########
+
+#. 
+   Install Intel(R) oneAPI DPC++ Compiler (select the variant as described in
+   :ref:`Selecting a Compiler<selecting_a_compiler>`).
+
+#. 
+   Clone this project to ``<path to onemkl>``\ , where ``<path to onemkl>``
+   is the root directory of this repository.
```

Comment:
In my opinion, "clone the project to `dir`" **makes** `dir` the root directory of the repository, so maybe we can say something like
>Clone this project. The root directory of the cloned repository will be referred to as `<path to onemkl>`.

Reply:
Done in [9219af9](https://github.com/oneapi-src/oneMKL/pull/510/commits/9219af9d107edd333a04cbefda7b3d816176695c).